faq.html

```diff
@@ -94,17 +64,26 @@
       </figcaption>
     </figure>
 
-    <p>
-      You'll need arrows and a quiver suitable for horse archery. You will probably find you need gloves and/or a thumb ring.
-    </p>
+    <ol>
+      <li><p><strong>A bow</strong>. Specifically, you'll want a traditional asiatic recurve bow, a.k.a composite bow, a.k.a  "horse bow". Modern features such as sights, arrow-rests, pulleys, and pistol grips get in the way at the speeds a horse archer. Plus a shorter bow (or an asymmetrical one like the Japanese yumi) tends to be easier to manage in the saddle. We shoot many arrows at relatively short distances, so high poundage bows are unnecessarily fatiguing even at the highest levels of the sport.</p>
```

Comment:
get in the way at the speeds [of] a horse archer

Reply:
done!